app/javascript/react/components/Calendar/Month.style.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+
+import { grey100, grey400 } from "../../assets/styles/colors";
+
+interface dayProps {
+  color?: string;
+  shouldColor: boolean;
+  isCurrentMonth: boolean;
+}
+
+const Day = styled.div<dayProps>`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 50px;
+  height: 50px;
+  padding: 4px;
+  opacity: 0.8;
+  background-color: ${(props) =>
+    props.shouldColor ? props.color : "transparent"};
+  border-radius: 5px;
+`;
+
+interface labelProps {
+  shouldDisplay: boolean;
+  isGrayedOut?: boolean;
+}
+
+const DayNumber = styled.label<labelProps>`
```

Comment:
Label is more natural with an input element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label
I'd go with a simple span here.

Reply:
That's interesting, thanks! I used `label` habitually, in iOS label is used for displaying text :D